easybuild/easyconfigs/n/netCDF/netCDF-4.4.1.1-CrayGNU-17.08.eb

```diff
@@ -0,0 +1,39 @@
+name = 'netCDF'
```

Comment:
see comments in https://github.com/eth-cscs/production/pull/334/files

Reply:
Done.